packages/server-wallet/src/message-service/types.ts

```diff
@@ -0,0 +1,35 @@
+import {Message} from '@statechannels/client-api-schema';
+
+/**
+ * This is the handler that any messaging service implementation should call when receiving a message
+ * The handler is responsible for pushing the message into the appropriate wallet
+ */
+export type MessageHandler = (
+  // Including the to makes it easier for a messaging service to handle multiple wallets if it chooses to do so
+  to: string,
+  message: unknown,
```

Comment:
```suggestion
  message: Message
```
Probably an improvement with respect to code re-use and type consistency.

Reply:
Resolved in 0accce4